ibm_cloud_sdk_core/authenticators/bearer_token_authenticator.py

```diff
@@ -15,9 +15,11 @@
 # limitations under the License.
 
 from requests import Request
-
```

Comment:
This empty line should be there to separate 3rd party and local package imports. :)

Reply:
fixed